solidity/slither.config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "exclude_dependencies": true,
+    "filter_paths": "test"
```

Comment:
This should be `contracts/test`, no? Seeing we have a test fix as well hehe.

Reply:
Yeah, `contracts/test` is better. I started fixing stubs, then realized I can exclude them and because the history was a mess I started rebasing things. So it's not clear from the commit history, but the order of things was different and I added `filter_paths` at the end 😆